site/docs/patterns/international-address-input.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: International Address Input
+layout: DetailPattern
+sidebar:
+  label: International Address Input
+aliases:
+  - /salt/patterns/international-address-input
+---
+
+International Address Input is a pattern that gives users a set of form fields to input address information for specific countries.
+
+## Pattern scope
+
+The pattern is a form that displays relevant fields for a user-selected location. You can use this pattern to provide a consistent look and feel when implementing International Address Input interactions across your applications. Future versions of the pattern will cover use of APIs, use of autocompletion, and enhanced support for more locations.
+
+The pattern aims to accommodate users worldwide, but your team may be able to take further steps that fall outside the scope of this pattern. For example, check with your tech partners if it is possible to detect user location and automatically adjust the form.
+
+## When to use it
+
+Use the pattern when your user needs to enter an address for a specific country. This could be for shipping, contact, legal or compliance purposes, such as for a client onboarding flow or client detail collection process.
+
+## Pattern content
+
+The pattern consists of Salt Form Field components (for layout guidance, please refer to the [Forms pattern](./forms)). It comprises two formats: Dynamic and Generic.
+
+### Dynamic format
+
+Dynamic format generates templates based on country or region. The templates only display fields that are relevant to a specific country, in a recognizable order, and with familiar labels.
+
+For the initial release, Dynamic format supports five countries: France, Germany, Spain, the United Kingdom and the United States.
+
+We recommend using templates in Dynamic format with localized language and terms that are more familiar to the user. This will reduce barriers to input and minimize user errors.
+
+<ImageSwitcher
+  images={[
+    {
+      src: "/img/patterns/international-address-input/select_country.png",
+      alt: "Example showing a Select country dropdown",
+    },
+    {
+      src: "/img/patterns/international-address-input/select_united_states.png",
+      alt: "Example showing Dynamic template with United States selected",
+    },
+  ]}
+  label="Select United States"
+/>
+
+<Diagrams>
+  <Diagram
+    src="/img/patterns/international-address-input/dynamic_france.png"
```

Comment:
These images doesn't feel high res on Mac. Try 2x export in Figma.

Reply:
2x makes them bigger in the site but the same not-high-res quality. These are the same quality as the existing Forms Pattern page, but I can look into other ways to improve this.